Infrastructure/PluginExecutor.cs

```diff
@@ -31,8 +32,13 @@ public PluginExecutor(IMessageSender messageSender, IDal dal, PluginsMenu plugin
             _pluginsMenu = pluginsMenu;
             _pluginsManager = pluginsManager;
         }
+        public string PluginId()
```

Comment:
This function is not in use, you can remove it

Reply:
ok. It was one of the attempts along the way.. I just forgot to delete.